index.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+      <meta charset="UTF-8">
+      <meta name="viewport" content="width=device-width, initial-scale=1.0">
+      <title>Project</title>
+        <link rel="stylesheet" href="style.css">
+    </head>
+    <body>
+        <header class ="header"><h3>Hello Microverse!</h3></header>
```

Comment:
Kindly replace `<h3>` tag with `<h1>` tag.  The `<h1>` tag should describe the topic of your page and its content. Normally, the `<h1>` tag gives the reader an idea of the content of a web page. For more info please visit this [Link](https://smartkeyword.io/en/seo-on-page-h1-tag/)